lib/src/dotenv.dart

```diff
@@ -60,21 +60,29 @@ class DotEnv {
   /// Loads environment variables from the env file into a map
   /// Merge with any entries defined in [mergeWith]
   Future<void> load(
-      {String fileName = '.env', Parser parser = const Parser(), Map<String, String> mergeWith = const {}}) async {
+      {String fileName = '.env',
+      Parser parser = const Parser(),
+      Map<String, String> mergeWith = const {}}) async {
     clean();
     final linesFromFile = await _getEntriesFromFile(fileName);
-    final linesFromMergeWith = mergeWith.entries.map((entry) => "${entry.key}=${entry.value}").toList();
+    final linesFromMergeWith = mergeWith.entries
+        .map((entry) => "${entry.key}=${entry.value}")
+        .toList();
     final allLines = linesFromMergeWith..addAll(linesFromFile);
     final envEntries = parser.parse(allLines);
     _envMap.addAll(envEntries);
     _isInitialized = true;
   }
 
   void testLoad(
-      {String fileInput = '', Parser parser = const Parser(), Map<String, String> mergeWith = const {}}) {
```

Comment:
What's caused these formatting changes throughout? Is this an IDE preference? Can we revert formatting changes for the sake of reviewing the diff? Unless these formatting options are from the analyser.